### DISCUSSION TITLE
Verified badge

### DISCUSSION THREAD

**Discussion by Kcjay2068** [community/community#62341] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I want my my Instagram,Facebook and Twitter account to be verified how do I get that ?

**Comment by Mittelblut9** [comment]
This seems very off topic. To verify your social media accounts, please contact their support.

Else, as in my knowledge you can’t verify your social media here on GitHub.

**Comment by h7x4** [comment]
I don't think you can "verify" your accounts on GitHub per se, but you can definitely link them!

If you go to https://github.com/settings/profile, you can add your profiles here:

![image](https://github.com/community/community/assets/14929991/b831652d-64c3-4d46-a60a-b4d304b34111)

**Comment by queenofcorgis** [comment]
Hi @Kcjay2068,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on those sites. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
